.travis.yml

```diff
@@ -17,7 +17,7 @@ matrix:
       script: Rscript -e 'pkgdown::deploy_site_github()'
       skip_cleanup: true
       on:
-        condition: $TRAVIS_BRANCH = master || -n $TRAVIS_TAG
+        condition: $TRAVIS_REPO_SLUG = r-lib/fs && $TRAVIS_BRANCH = master || -n $TRAVIS_TAG
```

Comment:
Looks like travis provides a [shorthand](https://docs.travis-ci.com/user/deployment#conditional-releases-with-on) for this, so you should be able to use

```yaml
 on:
   repo: r-lib/fs
```

Reply:
Updated!